src/node_api.cc

```diff
@@ -22,6 +23,10 @@ node_napi_env__::node_napi_env__(v8::Local<v8::Context> context,
   CHECK_NOT_NULL(node_env());
 }
 
+node_napi_env__::~node_napi_env__() {
+  FinalizeAll();
```

Comment:
This is already part of `~napi_env__()` – is there a reason for that?

Reply:
`CallFinalizer` is a virtual method and should not be called from the base constructor's destructor. I'd think I need to remove the `FinalizeAll` in `~napi_env__()`.